ui/src/test/org/jboss/windup/web/selenium/Selenium01Test.java

```diff
@@ -0,0 +1,198 @@
+package org.jboss.windup.web.selenium;
+
+import java.awt.AWTException;
+import junit.framework.TestCase;
+
+/**
+ * Runs the first test from the Web UI Test Script V0.1
+ * @author elise
+ */
+public class Selenium01Test extends TestCase {
+
+	private CreateProject selenium;
+
+	public void setUp() {
+		selenium = new CreateProject();
+	}
+
+	/**
+	 * Tests the first and second step
+	 */
+	public void testStep01_02() {
+		/*
+		 * Step 01
+		 */
+		assertEquals("http://127.0.0.1:8080/rhamt-web/project-list", selenium.checkURL());
+		
+		selenium.clickNewProjButton();
+		assertEquals("http://127.0.0.1:8080/rhamt-web/wizard/create-project", selenium.checkURL());
+		
+		//checks that the project name field has focus, then the cancel/next buttons are enabled/disabled
+		//.nameInputSelected does not work
+		assertTrue(selenium.nameInputSelected());
+		assertTrue(selenium.cancelEnabled());
+		assertFalse(selenium.nextEnabled());
+		
+		/*
+		 * step 02
+		 */
+		// have to click cancel twice to get out of the name input
```

Comment:
actually this should not be the case, right? Is that an error in the UI? then it should fail instead and we should log a ticket?

Reply:
Yes of course - https://issues.jboss.org/browse/WINDUP-2154